runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -327,6 +327,108 @@ void J9::RecognizedCallTransformer::process_java_lang_StringUTF16_toBytes(TR::Tr
    treetop->insertAfter(TR::TreeTop::create(comp(), TR::Node::create(node, TR::treetop, 1, newCallNode)));
    }
 
+/*
```

Comment:
Just a suggestion, but this detailed comment should go to header file for the function (Similar to how it is done for other `process` functions in same file